lib/alice/earmuffs.ex

```diff
@@ -6,7 +6,7 @@ defmodule Alice.Earmuffs do
   use Alice.Router
   alias Alice.Conn
 
-  command ~r/>:? earmuffs\b/i, :earmuffs
+  command(~r/>:? earmuffs\b/i, :earmuffs)
```

Comment:
This is bad, we need to add something to ignore `command` and `route`.

Reply:
@adamzaninovich good catch!

I added `locals_without_parens` to the .formatter so it will ignore this DSL for `route/2` and `command/2`. I also exported this formatting option so that other third party alice handlers can import this and expect the same formatting rules for their handlers.

Reply:
The last todo I think is to probably update the README for that formatting option

Reply:
oh whoops... we need like a WIP label or something.

Reply:
I think adding CI with `credo` and leaving `#TODO`'s in working branches that will fail CI, and then having github now allow merges for failed CI branches would help